tools/operator/testclient/operations/invokequery.go

```diff
@@ -137,6 +138,11 @@ func (i InvokeQueryUIObject) InvokeQuery(config inputStructs.Config, tls, action
 	if action == "Query" {
 		configObjects = config.Query
 	}
+
+	// print action in bold
+	fmt.Printf("\n\033[1m"+"action:%s\n"+"input:\n"+"\033[0m", action)
+	spew.Dump(configObjects)
+
```

Comment:
While in theory, it shouldn't matter that these are one after another, if you are doing commands in parallel (and its not unreasonable for query and invoke that you might be running multiple instance of the operator, thats what we did for load testing), aggregating the logs together could return results where the first line and the second line are not together in the log. You get around this by using `spew.Sdump` to grab the stringified output: fmt.Printf("\n\033[1m"+"action:%s\n"+"input:%s\n"+"\033[0m", action, spew.Sdump(configObjects))